config/bmo/ca_secret.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+data:
+  ca.crt: ${IRONIC_CA_CERT_B64:=""}
```

Comment:
do we need the '='

Reply:
yes, to default to empty string. This way, the user does not have to set this variable